### PR TITLE
GROW-130 Featured Loans

### PR DESCRIPTION
**Pull request by eddieferrer**
* Implemented features loans on the lend by category homepage

### CODE REVIEW

File: src/components/LoanCards/LendHomepageLoanCard.vue [hunk 1]
```diff
@@ -194,15 +194,15 @@ $card-half-space: rem-calc(14/2);
 	box-shadow: 0 0.65rem $card-margin $card-half-space rgb(153, 153, 153, 0.1);
 
 	&__image-wrapper {
-		height: rem-calc(168);
+		height: rem-calc(165);
```
Comment (eddieferrer) [inline src/components/LoanCards/LendHomepageLoanCard.vue:197]: I made some very minor style tweaks here.

File: src/pages/Homepage/LendByCategoryHomepage.vue [hunk 2]
```diff
@@ -335,11 +337,27 @@ export default {
...
+		padding: 6rem 0;
+	}
+}
+
 .lend-by-category-homepage {
 	overflow: hidden;
 }
 
 .featured-loans {
+	padding: 2rem 0 1rem;
```
Comment (eddieferrer) [inline src/pages/Homepage/LendByCategoryHomepage.vue:355]: This section and `.loan-categories` dont have the same padding as per the design

**Comment by eddieferrer** [comment]
This PR has the basic functionality of the featured loans carousel. 

Now that we have preFetch working again on homepage subcomponents. We can prefetch the categories setting, then prefetch the category information, and the featured loan for the first category. 

Issues I'm aware of and will followup on. 

- Not currently filtering funded loans based on fundedAmount/reservedAmount, I suppose if a funded loan comes back as the featured loan, we should fetch another loan. I wonder if this is worth doing, as it would be 2 sequential API requests, and that could take longer than the 5 seconds the slide takes to auto slide. Maybe we just show a funded loan? I think we should consider filtering funded loans on the backend and passing that as a param so that they are not returned. 
- Currently the featured loans carousel and the first loan on the category carousel will be the same. I will fix this in a followup PR.

**Comment by emuvente** [comment]
> Not currently filtering funded loans based on fundedAmount/reservedAmount

This could also be handled by requesting 3-4 loans in the initial query and displaying the first one that isn't funded, which should work for most cases. For the rare case where all of the loans from the initial query are funded, a second query could be done at that point to get another 3-4 loans.

Handling it on the backend is an option too, but we would have to figure out what to do with the limit and offset parameters.

**Pull request merged**